src/Controller/Async/General.php

```diff
@@ -28,8 +28,8 @@ protected function addRoutes(ControllerCollection $c)
         $c->get('/', 'async')
             ->bind('async');
 
-        $c->get('/changelog/{contenttype}/{contentid}', 'changeLogRecord')
-            ->value('contenttype', '')
+        $c->get('/changelog/{contenttypeslug}/{contentid}', 'changeLogRecord')
+            ->value('contenttypeslug', '')
```

Comment:
BC bread

Reply:
How come? It's only used in one place, and i'm not actually changing what's passed in, just what's done with it.

Reply:
It changes the router signature